src/identity/components/GlobalHeader/OrgDropdown.tsx

```diff
@@ -24,12 +24,12 @@ export const OrgDropdown: FC<Props> = ({activeOrg, orgsList}) => {
   const orgMainMenu = [
     {
       name: 'Settings',
-      iconFont: IconFont.CogOutline,
+      iconFont: IconFont.CogSolid_New,
       href: `/orgs/${activeOrg.id}/about`,
     },
     {
       name: 'Members',
-      iconFont: IconFont.UserOutline_New,
+      iconFont: IconFont.Group,
```

Comment:
Looking at the Figma I see why this makes sense. Flagging for @juliajames12 since this is part of globalheader.